pkgs/development/python-modules/azure-cosmos/default.nix

```
@@ -6,12 +6,12 @@
 }:
 
 buildPythonPackage rec {
-  version = "3.2.0";
+  version = "4.3.0";
   pname = "azure-cosmos";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "4f77cc558fecffac04377ba758ac4e23f076dc1c54e2cf2515f85bc15cbde5c6";
+    sha256 = "";
```

```suggestion
    sha256 = "";
```

Does not seem right. Please check hashes.

---

nix-update messed it up, it occasionally leaves these empty hashes. Let me fix it.